app/presenters/verify_documents_response_presenter.rb

```diff
@@ -0,0 +1,32 @@
+class VerifyDocumentsResponsePresenter
+  include Rails.application.routes.url_helpers
+
+  def initialize(form:, form_response:)
+    @form = form
+    @form_response = form_response
+  end
+
+  def success
+    @form_response.success?
+  end
+
+  def errors
+    @form_response.errors.flat_map do |key, errs|
```

Comment:
this is the exact same as `ImageUploadResponsePresenter` --- can we just reuse the class?

Reply:
wasn't sure if i was going to need to make mods... i guess i can change it if i do